ZukoAzula2016/src/org/team1540/zukoazula/Shooter.java

```diff
@@ -13,18 +15,20 @@
 import ccre.frc.FRC;
 import ccre.time.Time;
 
-public class ShooterSpeedControl {
-    private static final ExtendedMotor shooterCAN = FRC.talonCAN(7);
-    private static BooleanInput shooting;
-    private static FloatOutput shooterMotor;
-    private static FloatInput gearTooth;
+public class Shooter {
+    private static final ExtendedMotor shooterLeftCAN = FRC.talonCAN(10);
+    private static final ExtendedMotor shooterRightCAN = FRC.talonCAN(11);
+    private static final FloatInput encoder = FRC.encoder(0, 1, false, FRC.startTele);
+
+    private static final ArbitratedBoolean shooterTrigger = ZukoAzula.behaviors.addBoolean();
 
     public static void setup() throws ExtendedMotorFailureException {
-        shooting = ZukoAzula.controlBinding.addBoolean("Shoot");
-        shooterMotor = shooterCAN.simpleControl();
-        gearTooth = FRC.counter(0, 1, FRC.startTele);
-        bangBangControl(gearTooth, shooting, ZukoAzula.mainTuning.getFloat("Shooter Velocity Target", 1), 0, 1).send(shooterMotor);
-        Cluck.publish("Shooter Motor", shooterMotor);
+        BooleanInput trigger = ZukoAzula.controlBinding.addBoolean("Shoot");
+        shooterTrigger.attach(ZukoAzula.teleop, trigger);
+        shooterTrigger.attach(ZukoAzula.pit, trigger);
```

Comment:
Not currently, but that sounds like a feature request for the CCRE! --> https://github.com/flamingchickens1540/common-chicken-runtime-engine/issues